lib/business/janitor.dart

```diff
@@ -0,0 +1,73 @@
+import '../models/tasks.dart';
+import '../utils/io.dart';
+
+class Janitor {
+  late List<JanitorTask> janitorTasks;
+
+  Janitor() {
+    refresh();
+  }
+
+  Future<bool> editStatus(TaskStatus status, int taskIndex) async {
+    if (taskIndex >= janitorTasks.length || taskIndex < 0) return false;
+
+    janitorTasks[taskIndex].status = status;
+
+    var io = IO();
+    var parameter = <String, String>{'request_id': taskIndex.toString()};
```

Comment:
Az API paramétere helyesen az `uuid` névre hallgat. Ezt pedig a megfelelő indexű feladat `uid` mezője tartalmazza, mint ahogy a delete függvénynél helyesen is írtátok:)
```suggestion
    var parameter = <String, String>{'uuid': janitorTasks[taskIndex].uid};
```